book/doctrine.rst

```diff
@@ -916,8 +924,81 @@ a ``name`` field and the associated getter and setter functions.
 Relationship Mapping Metadata
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To relate the ``Category`` and ``Product`` entities, start by creating a
-``products`` property on the ``Category`` class:
+In our example, each category can be associated with *many* products, while
```

Comment:
One last comment: In the docs we never used the first person perspective. So I would change this to "In this example, [...]".